app/kubernetes-ingress-controller/migrate/ingress-to-gateway.md

```diff
@@ -21,16 +21,40 @@ tags:
 related_resources:
   - text: Ingress to Gateway FAQ
     url: /kubernetes-ingress-controller/faq/migrate-ingress-to-gateway/
+  - text: KongIngress Migration
+    url: /kubernetes-ingress-controller/migrate/kongingress/
 
 ---
 
+{:.warning}
```

Comment:
I remember we need to enable `GatewayAlpha` feature gate in KIC to support `TCPRoute`, `UDPRoute` and `TLSRoute`. Should we mention it in the migration docs?

Reply:
Thank you!

Reply:
Added here https://deploy-preview-2258--kongdeveloper.netlify.app/kubernetes-ingress-controller/migrate/ingress-to-gateway/#prerequisites